Tasks/AzureRmWebAppDeploymentV4/Tests/WindowsWebAppZipDeployProviderL0Tests.ts

```diff
@@ -0,0 +1,76 @@
+import tl = require('vsts-task-lib');
+import { DeploymentFactory } from '../deploymentProvider/DeploymentFactory';
+import { WindowsWebAppZipDeployProvider } from '../deploymentProvider/WindowsWebAppZipDeployProvider'
+import { IWebAppDeploymentProvider } from '../deploymentProvider/IWebAppDeploymentProvider';
+import { TaskParametersUtility, TaskParameters, DeploymentType } from '../operations/TaskParameters';
+import { stringify } from 'querystring';
+import { PackageType } from 'webdeployment-common/packageUtility';
+import { getMockEndpoint } from '../node_modules/azure-arm-rest/Tests/mock_utils';
+import { mockAzureARMPreDeploymentSteps, mockZipDeploySettings }  from "./mock_utils";
+
+getMockEndpoint();
+mockAzureARMPreDeploymentSteps();
+mockZipDeploySettings();
+
+export class WindowsWebAppZipDeployProviderL0Tests  {
+
+    public static async startWindowsWebAppZipDeployProviderL0Tests() {
+        await WindowsWebAppZipDeployProviderL0Tests.testForPreDeploymentSteps_ZipDeployProvider();
+        await WindowsWebAppZipDeployProviderL0Tests.testForPreDeploymentStepsWithSlotEnabled_ZipDeployProvider();
```

Comment:
add for folder package type

Reply:
Done. Added in this commit fdd57b62de16f1d6f2671cd79cf1988304381151